flake8_eradicate.py

```diff
@@ -43,9 +33,23 @@ def __init__(self, physical_line, tokens) -> None:
         """
         self._physical_line = physical_line
         self._tokens = tokens
-        self._options = _Options(
-            aggressive=self.options.eradicate_aggressive,  # type: ignore
-        )
+        self._options = {
+            'aggressive': self.options.eradicate_aggressive,  # type: ignore
+        }
+
+        self.eradicator = Eradicator()
```

Comment:
Do we need this property to be public and not protected?

Reply:
Made it protected.